Add annotation to ntrans.py and add --strict flag to github actions

ThePhilgrim opened the pull request.

ThePhilgrim committed:
Add annotation to ntrans.py and add --strict flag to github actions

ThePhilgrim committed:
Add ntrans_combine to black check

ThePhilgrim committed:
Black code

Akuli commented on .github/workflows/github_actions.yml:
```diff
@@ -11,7 +11,7 @@ jobs:
         python3 -m pip install black
     - run: |
-        python3 -m black --check ntrans.py ntrans_dataprep.py
+        python3 -m black --check ntrans.py ntrans_dataprep.py ntrans_combine.py
```
It seems like we always want to black all python files, so we might as well specify that instead of listing file names:

```suggestion
        python3 -m black --check *.py
```

Akuli commented on .github/workflows/github_actions.yml:
```diff
@@ -22,4 +22,4 @@ jobs:
         python3 -m pip install mypy
     - run: |
-        python3 -m mypy ntrans.py ntrans_dataprep.py
+        python3 -m mypy --strict ntrans.py ntrans_dataprep.py ntrans_combine.py
```
Same here:

```suggestion
        python3 -m mypy --strict *.py
```

ThePhilgrim committed:
Update .github/workflows/github_actions.yml

Ah, I didn't know that this could be done! Great.

Co-authored-by: Akuli <akuviljanen17@gmail.com>

ThePhilgrim committed:
Update .github/workflows/github_actions.yml

Even after running mypy --strict on ntrans_combine, it says everything looks good. So I assume I didn't forget anything?
Or is there another flag that checks missing annotation specifically?

Co-authored-by: Akuli <akuviljanen17@gmail.com>

ThePhilgrim commented:
@Akuli Please see comment on second suggested change

Akuli commented:
Can't see it for whatever reason. Can you copy it here?

Akuli commented:
Ah, now I see the comment. No, you didn't forget anything.
